applications/nrf_desktop/src/modules/ble_scan.c

```diff
@@ -221,6 +222,52 @@ static int configure_filters(void)
 	return err;
 }
 
+static bool is_llpm_peer_connected(void)
+{
+	bool llpm_peer_connected = false;
+
+	for (size_t i = 0; i < ARRAY_SIZE(subscribed_peers); i++) {
+		if (!bt_addr_le_cmp(&subscribed_peers[i].addr, BT_ADDR_LE_NONE)) {
+			break;
+		}
+
+		struct bt_conn *conn = bt_conn_lookup_addr_le(BT_ID_DEFAULT,
+						&subscribed_peers[i].addr);
+
+		if (conn) {
+			bt_conn_unref(conn);
+			if (subscribed_peers[i].llpm_support) {
+				llpm_peer_connected = true;
+				break;
+			}
+		}
+	}
+
+	return llpm_peer_connected;
+}
+
+static void update_init_conn_params(bool llpm_peer_connected)
+{
+	struct bt_le_conn_param cp = {
+		.latency = 0,
+		.timeout = 400,
+	};
+
+	/* In case LLPM peer is already connected, the next peer has to be
```

Comment:
btw - what will happen if we first have a BLE client and then LLPM one? It should lead to similar problems, wont it?

Reply:
I checked and it looks ok. LLPM will get over 500 reports/second.
Maybe it could become even better after the API update (I am unable to test it now).